.github/jobs/macos_cmake.yml

```
@@ -0,0 +1,15 @@
+parameters:
+  vmImage: ''
+
+steps:
+  - script: |
+      wget https://github.com/Kitware/CMake/releases/download/v$(CMAKE_VERSION)/cmake-$(CMAKE_VERSION)-macos-universal.tar.gz
```

I noticed that my dev machine does not have wget, adding some snippets so all our bases are covered for installation and reinstallation, unless we're already doing this in a previous step.
```suggestion
        brew install wget
        wget https://github.com/Kitware/CMake/releases/download/v$(CMAKE_VERSION)/cmake-$(CMAKE_VERSION)-macos-universal.tar.gz
```

---

wget is available by default on all VMs. Do it locally only.

---

cool, not needed.